understanding/non-text-content.html

```diff
@@ -263,180 +227,106 @@ <h3>
-                     modality (for example, visual, auditory or tactile).  Short and long text alternatives
-                     can be used as needed to convey the information in the non-text content.  Note that
-                     
-                     <strong>
-                        prerecorded
-                        audio-only
-                        </strong> and 
-                     
-                     <strong> prerecorded
-                        video-only
-                        </strong> files are covered here.  
-                     <strong>
-                        Live-audio-only
-                        </strong> and 
-                     <strong>
-                        Live-video-only
-                        </strong> files are covered below (see 3rd paragraph following this one).  
+                     <strong>下記のその他の状況のいずれも適用されない非テキストコンテンツ</strong> (例: グラフ、図表、録音した音声、写真、及びアニメーション) の場合、テキストによる代替はあらゆる感覚モダリティ (例えば、視覚、聴覚、又は触覚) によってレンダリング可能な形態で同じ情報を入手可能にできる。短い、及び長いテキストによる代替は、非テキストコンテンツにある情報を伝達するのに必要に応じて用いられる。<strong>収録済の音声しか含まないファイル</strong>及び<strong>収録済の映像しか含まないファイル</strong>は、ここで扱われている。<strong>ライブの音声しか含まない</strong>ファイル及び<strong>ライブの映像しか含まない</strong>ファイルは、以下で扱われている (この段落の三つ後の段落を参照のこと)。  
                      
                   </p>
                   
                   
                   <p> 
                      
-                     <strong>
-                        For non-text content that is a control or accepts user input
-                        </strong>, such as images used as submit buttons, image maps or complex animations, a name
-                     is provided to describe the purpose of the non-text content so that the person at
-                     least knows what the non-text content is and why it is there.
+                     <strong>コントロール又は利用者の入力を受け入れる非テキストコンテンツ</strong>、例えば、送信ボタンとして用いられる画像、イメージマップ、又は複雑なアニメーションなどの場合、名前は、少なくともその非テキストコンテンツが何で、なぜそこにあるのかを認識できるように、その非テキストコンテンツの目的を説明するために提供される。
                      
                   </p>
                   
                   
                   <p> 
                      
-                     <strong>
-                        Non-text content that is time-based media
-                        </strong>
-                     is made accessible through 
-                     <a href="time-based-media">1.2: Time-Based Media</a>.  However, it is important that users know what it is when they encounter it on a
-                     page so they can decide what action if any they want to take with it.  A text alternative
-                     that describes the time-based media and/or gives its title is therefore provided.
+                     <strong>時間依存のメディアである非テキストコンテンツ</strong>は、<a href="time-based-media.html">1.2: Time-Based Media</a> によりアクセシブルになる。しかし、利用者がそのメディアを利用したいことがもしあれば、利用者がどんな動作をするかを解決できるので、利用者がページ上でそのメディアに出会ったときに、そのメディアが何であるかを知ることが重要である。したがって、時間依存のメディアを説明する、及び／又はそのメディアのタイトルを示すテキストによる代替を提供する。
```

Comment:
解説書内のリンクタイトル（ガイドライン名）は翻訳しておきましょう
```suggestion
                     <strong>時間依存のメディアである非テキストコンテンツ</strong>は、<a href="time-based-media.html">1.2: 時間依存メディア</a>によりアクセシブルになる。しかし、利用者がそのメディアを利用したいことがもしあれば、利用者がどんな動作をするかを解決できるので、利用者がページ上でそのメディアに出会ったときに、そのメディアが何であるかを知ることが重要である。したがって、時間依存のメディアを説明する、及び／又はそのメディアのタイトルを示すテキストによる代替を提供する。
```